Lib/fontTools/varLib/instancer/names.py

```diff
@@ -83,6 +83,7 @@ def updateNameTable(varfont, axisLimits):
     # AxisValues found in step 1. The updated name table will conform to the
     # RIBBI naming model, more info can be found here:
```

Comment:
nit: you could insert that `(Regular, Italic, Bold, Bold Italic)` right after you first mention the acronym

Reply:
or even better, in the docstring where you talk about the `R/I/B/BI naming model`. docstrings are user visible, more than inline comments which are for devs

Reply:
actually the whole comment and example is now short enough that it could be moved to the docstring itself

Reply:
Thanks!